Pharmacy.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "Medicine.h"
+#include <vector>
+
+using namespace std;
+
+class Pharmacy
+{
+private:
+	vector<Medicine> l;
+	int* convert(string date);
+public:
+	Pharmacy(int n);
+	void discount();
+	void check(string date);
```

Comment:
Names should be more informative. (check_something)

Reply:
I changes names into check_date